src/native/config/PlatformHelper.android.ts

```diff
@@ -2,6 +2,7 @@ import { BaseItemAnimator } from "recyclerlistview";
 
 const PlatformConfig = {
   defaultDrawDistance: 250,
+  invertedTransformStyle: { transform: [{ rotate: "180deg" }] },
```

Comment:
Maybe we can add a comment for why we're doing this and link to the original issues. Might be surprising to see this down the road.